src/pykeen/nn/init.py

```diff
@@ -124,11 +122,32 @@ def create_init_from_pretrained(
 
     :return:
         an initializer, which fills a tensor with the given weights.
+
+    Added in https://github.com/pykeen/pykeen/pull/638.
```

Comment:
do we want to keep this info? seems to blow up the docstring (and can be seen by `git blame` anyway)

Reply:
I think it would be nice to start linking back to the issue tracker for more context